### ISSUE TITLE
How should we label the diamond sizing control?

### ISSUE THREAD

**Issue by samreid** [phetsims/wave-interference#365]
From the design document:

> “Square Diameter” is probably not the best label for the slider, as it may be interpreted as the diagonal length. Perhaps “Square Side” is better?

![image](https://user-images.githubusercontent.com/679486/55337216-7698e700-545b-11e9-98b4-fbb2264b8ca5.png)

Throughout the code, I have named that variable as "diamond" and hence in the initial draft I used the label "Diamond Diameter", but it is a bit long and gets scaled down:

![image](https://user-images.githubusercontent.com/679486/55337264-8dd7d480-545b-11e9-9076-ea0da1c14dce.png)

How should we name this control?

Also, on reflection, I realized the "circle + diamond" scene introduces two new aspects at the same time: composition (by combining two shapes) and the rotation of the square.  I think it looks great, but wonder if it would be pedagogically more straightforward to do circle + square (not rotated).  I don't feel strongly but wanted to bring it up for discussion since it pertains to this issue.

One more point: I think having the diamond shape does make it easier to verbally distinguish between the scenes.

**Comment by arouinfar** [comment]
Thanks @samreid. I think diamond is certainly a better descriptor than square. However, I think "diameter" could still be problematic, as it may be interpreted as the diagonal length, which would be a factor of sqrt(2) larger than the side length. Perhaps "Diamond Side" would do the trick?

> Also, on reflection, I realized the "circle + diamond" scene introduces two new aspects at the same time: composition (by combining two shapes) and the rotation of the square. I think it looks great, but wonder if it would be pedagogically more straightforward to do circle + square (not rotated). I don't feel strongly but wanted to bring it up for discussion since it pertains to this issue.
>
>One more point: I think having the diamond shape does make it easier to verbally distinguish between the scenes.

Good points @samreid. I slightly prefer the diamond, but I recognize that it does have some baggage. Let's check in with the design team about whether it would be better to keep the diamond, or remove the rotation and go with the square.

**Comment by samreid** [comment]
We discussed options:

* diamond side
* side length
* diamond side length

We like to have the word "diamond" in the label, but it is a bit wordy.  We will go with "diamond side length" but will need to adjust the layout accordingly.

**Comment by samreid** [comment]
Proposed fix committed, @arouinfar can you please review?

**Comment by arouinfar** [comment]
I like it @samreid! I think we still need to decide on diamond vs. square in design meeting, so I'll leave this open.

**Comment by samreid** [comment]
@ariel-phet recommends to use the term "size" instead of "side length".  
@kathy-phet said what about "diamond side"
@ariel-phet said "diamond width" and make it the hypotenuse length.

@ariel-phet is fine with making it a square (not rotated) and calling it "Square width".
@samreid is on board with that
@kathy-phet is fine with that too.

**Comment by samreid** [comment]
In the preceding commit, I converted from diamond to square.  Over to @arouinfar for review and feedback.  Also, we increased the control panel widths to accommodate the long string "Diamond Side Length", do you want to reduce the control panel widths now that the labels are shorter?

**Comment by arouinfar** [comment]
@samreid looks good in master. Let's keep the increased panel width. It feels a bit less cramped, and has some room for i18n this way.

We'll also need to update the scene icon to be a square, so back to @samreid.

**Comment by samreid** [comment]
Good catch @arouinfar, ready for review on phettest.

**Comment by arouinfar** [comment]
👍
